src/react-app-env.d.ts

```diff
@@ -1 +1,51 @@
 /// <reference types="react-scripts" />
```

Comment:
You need to create folder types with one file for each interface.
Hint to discover(2): [https://mate-academy.github.io/fe-program/js/extra/react-typescript](url)